x-pack/plugins/security_solution/public/resolver/store/panel_view_and_parameters.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { decode } from 'rison-node';
+import { isPanelViewAndParameters } from '../models/location_search';
+import { PanelViewAndParameters } from '../types';
+import { parameterName } from './parameter_name';
+
+/**
+ * Return a value `PanelViewAndParameters` based on `locationSearch` and `resolverComponentInstanceID`.
+ * Used by selectors.
+ */
+export function panelViewAndParameters({
```

Comment:
This was previously found in the ui selector. This logic is needed by the data selectors as well, so it has been moved.